test/python/visualization/test_circuit_text_drawer.py

```diff
@@ -3925,7 +3927,7 @@ def test_draw_hamiltonian_multi(self):
             [
                 "      ┌──────────────┐",
                 "q0_0: ┤0             ├",
-                "      │  Hamiltonian │",
+                "      │  HAMILTONIAN │",
```

Comment:
I'm not sure this is an improvement. I guess it was inconsistent before between the drawers mpl was all lower case `hamiltonian` and latex and text were `Hamiltonian`. I think the previous behavior of Hamiltonian is probably better to preserve it looks weird as all caps as it's not an instruction like Initialize (although I think that's weird as all caps too).

Reply:
I'll take care of the release note.

On the capitalization issue, this is only really a text drawer issue. In the past, mpl and latex drawers had first letter caps, like 'Unitary', for all internal names, but mpl inadvertently lost it in #5360, so 'rcccx' became 'Rcccx' in latex and 'rcccx' in mpl. This PR puts mpl and latex back to being the same with first letter caps.

Prior to 0.14, the text drawer also had first letter caps, but in 0.14 this was changed to all caps, like 'RCCCX', 'RXX', etc., and it's been that way ever since. The change to 'HAMITONIAN' in this PR was just to make it consistent with 'RCCCX', etc. My preference is for the text drawer to be consistent with mpl and latex and go back to first letter caps only. If there's a decision on that, I can switch that back in this PR. I'd just have to change a few more text drawer tests.

Reply:
I agree lets make it consistent with the mpl/latex and only capitalize the first character. Right now it feels too much like old fortran code with all caps.

Reply:
Ha! My first language. Ok. I'll push the change and the release note shortly.

Reply:
Added reno and switched text drawer to first letter caps. https://github.com/Qiskit/qiskit-terra/pull/6078/commits/2d8bff8e9082b38e3a46ff3e94e68410a6d4400e

Reply:
@mtreinish I found a minor bug in the color selection and fixed it and added a test for it in the user_style mpl image test. Also fixed a capitalization issue in the text drawer after a main merge. Should be good to go now.